apps/api/infra/api.ts

```diff
@@ -155,6 +156,10 @@ export const serviceSetup = (services: {
       ISLYKILL_SERVICE_PASSPHRASE: '/k8s/api/ISLYKILL_SERVICE_PASSPHRASE',
       ISLYKILL_SERVICE_BASEPATH: '/k8s/api/ISLYKILL_SERVICE_BASEPATH',
       IDENTITY_SERVER_CLIENT_SECRET: '/k8s/api/IDENTITY_SERVER_CLIENT_SECRET',
+      DRIVING_LICNESE_BOOK_XROAD_PATH:
+        '/k8s/api/DRIVING_LICNESE_BOOK_XROAD_PATH',
+      DRIVING_LICNESE_BOOK_USERNAME: '/k8s/api/DRIVING_LICNESE_BOOK_USERNAME',
+      DRIVING_LICNESE_BOOK_PASSWORD: '/k8s/api/DRIVING_LICNESE_BOOK_PASSWORD',
```

Comment:
We should go for an X-Road "module" like the ones below in the `.xroad` section.

Reply:
Done. Can you confirm?